linchpin/__init__.py

```diff
@@ -109,7 +108,6 @@ def setup_rundb(self):
                     else:
                         raise
```

Comment:
raise ?? It would be great if  we might have to raise a custom exception instead empty raise.

Reply:
This line hasn't been changed in the latest commit. Is there a new reason to point it out now? Last time it was changed was with commit 356e6af0 (2017-12-14).

If you are curious, however, in this case, if the error is unknown and we'd want to know the actual exception raised since it would be fatal. I could raise a LinchpinError, except then it would hide the actual exception.